### PR TITLE
Doc/definition of done doc

### LINKED ISSUE #16: Definition of Done
Create PR Template for Definition of Done

### PR DESCRIPTION
**Pull request by simonschuhmacher**
- Adds definition of Done as a PR template
  - Also add a reference to it on top-level README
- Adds some commands as a reference for kubernetes deployments
- Add CONTRIBUTING.md

Closes #16

### CODE REVIEW

**Comment by simonschuhmacher** [comment]
@triumphvalley pls review ✌🏼

File: README.md [hunk 1]
```diff
@@ -81,3 +86,12 @@ If a new Kubernetes environment needs to be set up, however, there are scripts i
...
 3. Deploy the database to the cluster using `helm install -f helm/saunah-database/values-staging.yaml saunah-database helm/saunah-database`. (Or use a different vlaues file, depending on the target of your deployment.) Make sure that the database depends on the secrets set in step 1. and 2., thus is need to be executed after these steps.
+
+
+## 👌🏼 Definition of Done
+The Definition of Done is automatically applied as the pull-request template. It can be found in [docs/pull_request_template.md](./docs/pull_request_template.md).
+
+
+## 📚 Further Documentation
+
+Fother documentation can be found in [docs/README.md](./docs/README.md).
```
Comment (triumphvalley) [inline README.md:97]: typo

File: docs/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  - [ ] JavaDoc has been added
+  - [ ] API documentation has been added via annotations (if applicable)
+  - [ ] No unavoidable code smells
+- Testing
+  - [ ] All existing tests pass
+  - [ ] New unit tests created according to [CONTRIBUTING.md](./CONTRIBUTING.md#-testing-strategy)
+  - [ ] New unit tests pass
+- SonarCloud
+  - [ ] Quality gate passed
+- Compatibility to frondend
```
Comment (triumphvalley) [inline docs/pull_request_template.md:29]: typo

**Review by triumphvalley — COMMENTED** [review]
nice! just little typos

**Review by triumphvalley — DISMISSED** [review]
👍🏻👍🏻

**Comment by simonschuhmacher** [comment]
Just merged in main, needs another review @triumphvalley ✌🏼

**Review by triumphvalley — APPROVED** [review]

**Pull request merged**